### ISSUE TITLE
"skip_full_destinations: true" seems to be not working

### ISSUE THREAD

**Issue by kzsolt4444** [swar/Swar-Chia-Plot-Manager#1148]
Hi!

I set the "skip_full_destinations" property to true, but it seems, it doesn't working, because it still wants to plot to a fully plotted destination disk.
<img width="649" alt="bug" src="https://user-images.githubusercontent.com/35074431/120934743-e40dc580-c6ff-11eb-82f7-d8e4a49d6863.png">

Am i doing something wrong, or is this a known bug?
Thanks!

**Comment by speeder2005** [comment]
same here. feels like the manager  cannot handle the forecasting correctly. for example, another thread scheduled a job to a drive but hasn't write the final file into the drive. that usage is not counted. 
If I add an already fully filled drive into the destination list, the manager can skip the drive correctly.

**Comment by BubbleTrouble14** [comment]
Yes have been experiencing the same issue. Would try and fix it if I had a little more time. As its very annoying when one does alot of plots and has to check if some are full and end up cancling some plots.

**Comment by kchang4** [comment]
yep can confirm this doesn't work in windows 10 as well.

**Comment by Sekaiji** [comment]
I confirm it doesn't work for me either on windows 10.
3 jobs, 1 temp disk on each job, 10 same destination drives for all the jobs.
It happened multiple times already. I had to kill the process and manually copy the plot file to another disk.

**Comment by bilalwaheedch** [comment]
Facing same issue, hope this get resolved soon. Always have to manually go back and stop the plots and remove files started in the same drive, which results in both jobs to get stuck since drive only has space (300GB) for a single plot.
